utils/shoppingActions.ts

```diff
@@ -35,7 +35,6 @@ export const KeyboardValueToActionMap = {
 export const ownerActions = [
   ShoppingActions.SEND,
   ShoppingActions.CONSUME,
-  ShoppingActions.DOWNLOAD,
```

Comment:
There are still some `ShoppingActions.DOWNLOAD` in the codebase, pls remove all of them.